src/mesh/data/gridfromfile.cxx

```diff
@@ -98,7 +104,13 @@ bool GridFile::get(Mesh *UNUSED(m), BoutReal &rval, const string &name) {
   if(!file->is_valid())
     throw BoutException("File cannot be read");
   
-  return file->read(&rval, name);
+  bool success = file->read(&rval, name);
+    if(success){
```

Comment:
According to [CONTRIBUTING.md](https://github.com/boutproject/BOUT-dev/blob/master/CONTRIBUTING.md#details) it should be:
```
    if (success) {
```
instead of
```
    if(success){
```
i.e. whitespace after the `if`, and before the `{`


You are right, I wouldn't suggest not using braces 👍